waterlog/src/Globals.js

```diff
@@ -1,4 +1,4 @@
 export const Globals = {
-	API_URL: 'https://api.iot.retrotest.co.za',
+	API_URL: 'https://localhost:44383',
```

Comment:
is this a mistake or you want it to go into Dev?

Reply:
fixed